HomeTask/HomeTask/TaskList/TaskListViewModel.swift

```diff
@@ -44,44 +45,27 @@ class TaskListViewModel: ObservableObject {
         
     }
     
-    func validateNewTask(forTitle title: String) -> Bool {
-        return title.count > 1
-        
-    }
-    
-    func confirmPressed() {
-        let newTask = TaskModel(title: titleNewTask)
-        
-        if validateNewTask(forTitle: newTask.title) {
-            self.tasks.append(newTask)
-            showingAddNewTaskView.toggle()
-            titleNewTask = ""
-            textPlaceholder = "Nome da atividade"
-            
-        } else {
-            textPlaceholder = "A atividade precisa ter um nome"
-            
+    func swipeAction(task: TaskItem) {
```

Comment:
Seguindo o padrão de design de APIs de Swift, isso poderia ser escrito de forma a indicar a própria ação. Ex:
```suggestion
    func didSwipe(on taskItem: TaskItem) {
```

Reply:
perfeito!